test/azure/low-level/AcceptanceTests/test_zzz.py

```diff
@@ -59,6 +59,7 @@ def test_ensure_coverage(self):
             "LROErrorPost202RetryInvalidHeader": 1,  # The reason this fails is bc the convenience layer fails in deserializing the retry-after header /bar as an int. LLC doesn't do that, so skipping
             "LROErrorPostAsyncInvalidHeader": 1,  # The reason this fails is bc the convenience layer fails in deserializing the retry-after header /bar as an int. LLC doesn't do that, so skipping
             "LROErrorPostAsyncInvalidJsonPolling": 1,   # The reason this fails is bc the convenience layer fails in deserializing. LLC doesn't do that, so skipping
+            "LROPatchInlineCompleteIgnoreHeaders": 1,
```

Comment:
The failure seems not caused by this PR but the change of: 
![image](https://user-images.githubusercontent.com/70930885/129524769-45d2612c-ea0f-4e6f-9caf-b9f9d1e7cb8a.png)

Reply:
@msyyc this is because a [test was added to the testserver](https://github.com/Azure/autorest.testserver/pull/298) that we don't have coverage for. The protocol here should be to add the test coverage for this in a separate PR, and then to continue with this PR. I've started a PR for this but haven't had the time to finish it, you can try looking into it, but I'll also keep looking into it tomorrow. Thanks!

Reply:
@msyyc turns out we expect this test to fail, the scenario is debatable. I'm going to create a PR to skip it everywhere, thanks!